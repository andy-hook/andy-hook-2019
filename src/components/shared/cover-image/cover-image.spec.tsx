import React from "react"
import renderer from "react-test-renderer"

import CoverImage from "./cover-image"
import { mockFluidImageObject } from "@mock-data"

describe("<CoverImage />", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<CoverImage imageObject={mockFluidImageObject} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
