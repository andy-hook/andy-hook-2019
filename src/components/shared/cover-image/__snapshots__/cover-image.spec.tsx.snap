// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CoverImage /> renders correctly 1`] = `
<div
  className=" gatsby-image-wrapper"
  style={
    Object {
      "height": "100%",
      "overflow": "hidden",
      "position": "relative",
    }
  }
>
  <div
    style={
      Object {
        "paddingBottom": "66.66666666666667%",
        "width": "100%",
      }
    }
  />
  <picture>
    <source
      sizes="(max-width: 1000px) 100vw, 1000px"
      srcSet="/static/4a7de89fcee5d7cd97c940895602fbe0/4d406/cover-image.jpg 250w,↵/static/4a7de89fcee5d7cd97c940895602fbe0/32ee9/cover-image.jpg 500w,↵/static/4a7de89fcee5d7cd97c940895602fbe0/2f7e7/cover-image.jpg"
    />
    <img
      alt=""
      loading="eager"
      onLoad={[Function]}
      sizes="(max-width: 1000px) 100vw, 1000px"
      src="/static/4a7de89fcee5d7cd97c940895602fbe0/2f7e7/cover-image.jpg"
      srcSet="/static/4a7de89fcee5d7cd97c940895602fbe0/4d406/cover-image.jpg 250w,↵/static/4a7de89fcee5d7cd97c940895602fbe0/32ee9/cover-image.jpg 500w,↵/static/4a7de89fcee5d7cd97c940895602fbe0/2f7e7/cover-image.jpg"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "objectFit": "cover",
          "objectPosition": "center",
          "opacity": 0,
          "position": "absolute",
          "top": 0,
          "transition": "opacity 500ms",
          "width": "100%",
        }
      }
    />
  </picture>
  <noscript
    dangerouslySetInnerHTML={
      Object {
        "__html": "<picture><source srcset=\\"/static/4a7de89fcee5d7cd97c940895602fbe0/4d406/cover-image.jpg 250w,↵/static/4a7de89fcee5d7cd97c940895602fbe0/32ee9/cover-image.jpg 500w,↵/static/4a7de89fcee5d7cd97c940895602fbe0/2f7e7/cover-image.jpg\\" sizes=\\"(max-width: 1000px) 100vw, 1000px\\" /><img loading=\\"eager\\" sizes=\\"(max-width: 1000px) 100vw, 1000px\\" srcset=\\"/static/4a7de89fcee5d7cd97c940895602fbe0/4d406/cover-image.jpg 250w,↵/static/4a7de89fcee5d7cd97c940895602fbe0/32ee9/cover-image.jpg 500w,↵/static/4a7de89fcee5d7cd97c940895602fbe0/2f7e7/cover-image.jpg\\" src=\\"/static/4a7de89fcee5d7cd97c940895602fbe0/2f7e7/cover-image.jpg\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
      }
    }
  />
</div>
`;
